### PR TITLE
(PC-16264) fix(signUp): prevent Honor button from bouncing when submitting Honor statement

### PR DESCRIPTION
**Pull request by yorickeando**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-16264

## Checklist

I have:

- [ ] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)

## Before

https://user-images.githubusercontent.com/44037911/183909784-27769ec5-814c-4e75-a2bc-36147a07c69b.mov

## After

https://user-images.githubusercontent.com/44037911/183909843-352bfec1-eb13-4df5-9b82-a6e2a9fcefd1.mov

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

**Review by Lucasbeneston — APPROVED** [review]
Cool ! Il y a d'autres endroits ou il pourrait y avoir le même problème ?

File: src/features/identityCheck/pages/confirmation/IdentityCheckHonor.tsx [hunk 1]
```diff
@@ -76,7 +75,7 @@ export const IdentityCheckHonor = () => {
               type="submit"
               onPress={postHonorStatement}
               wording={t`Valider et continuer`}
-              isLoading={isLoading}
+              isLoading={isLoading || isUserProfileLoading}
```
Comment (viconnex) [inline src/features/identityCheck/pages/confirmation/IdentityCheckHonor.tsx:78]: pour être plus explicite on peut renommer `isLoading` en `isPostingHonorStatement`

Comment (yorickeando) [reply src/features/identityCheck/pages/confirmation/IdentityCheckHonor.tsx:80]: bonne idée

**Review by viconnex — APPROVED** [review]

**Comment by yorickeando** [comment]
@Lucasbeneston  oui, et c'est traité dans le ticket suivant : https://passculture.atlassian.net/browse/PC-16265

**Pull request merged**